app/src/main/java/fr/free/nrw/commons/upload/ThumbnailsAdapter.java

```diff
@@ -85,18 +87,23 @@ public void bind(int position) {
             UploadableFile uploadableFile = uploadableFiles.get(position);
             Uri uri = uploadableFile.getMediaUri();
             background.setImageURI(Uri.fromFile(new File(String.valueOf(uri))));
+            GradientDrawable drawable = new GradientDrawable();
 
             if (position == callback.getCurrentSelectedFilePosition()) {
+                drawable.setShape(GradientDrawable.RECTANGLE);
+                drawable.setStroke(6, Color.parseColor("#6187A1"));
```

Comment:
Could you define this color in **colors.xml** file and use `getColor()` instead?

Reply:
Done 👍